spec/amp-shadow-doc.md

```diff
@@ -96,6 +96,42 @@ Both `AMP.attachShadowDoc` and `AMP.attachShadowDocAsStream` return a `ShadowDoc
 - `shadowDoc.close()` - closes the AMP document and frees the resources.
 
 
+## Shadow DOM API and polyfills
+
+AMP Shadow Docs rely heavily on [Shadow DOM API](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Shadow_DOM). This is a powerful and elegant API, part of Web Components family. It allows natural isolation between major parts of the page and, as such, an ideal tool for PWAs and AMP Shadow Docs.
+
+Shadow DOM is currently implemented in Chrome and newer Safari, so it has a solid modern support and definitely not ubiquitous. AMP Shadow Docs API polyfills the necessary parts of Shadow DOM.
```

Comment:
"solid modern support and definitely not ubiquitous"...so which is it? :) maybe reword to something like "Shadow DOM is currently only implemented in Chrome and newer versions of Safari, so in order to support all of your users, the AMP Shadow Docs API relies on polyfills for necessary parts of Shadow DOM."

Reply:
Done